Cargo.toml

```diff
@@ -35,6 +35,7 @@ byteorder = { version = "1.1", features = ["i128"]}
 cargo_metadata = { version = "0.8", optional = true }
 directories = { version = "2.0", optional = true }
 rustc_version = { version = "0.2.3", optional = true }
+getrandom = "0.1.10"
```

Comment:
getrandom docs say

> This is a low-level API. Most users should prefer using high-level random-number library like rand.

And we already have `rand` in our dependencies. There's probably an appropriate function in the `rand` crate for filling a buffer

Reply:
TBH I feel like `getrandom` is the better match here. We are not "most users", we want to model system-level randomness APIs. That is *exactly* what the getrandom crate does.